src/services/mail-service.yml

```diff
@@ -76,3 +76,33 @@ roomDeletionNotificationEmail.text:
 roomDeletionNotificationEmail.html:
   en: <p>Hello!<br/><br/>User <b>{ownerName}</b> has deleted the room <b>{roomName}</b>.
   de: <p>Hallo!<br/><br/>Der Benutzer <b>{ownerName}</b> hat den Raum <b>{roomName}</b> gelöscht.
+roomMemberRemovalNotificationEmail.subject:
+  en: Your membership to a room has ended
+  de: Ihre Mitgliedschaft in einem Raum wurde beendet
+roomMemberRemovalNotificationEmail.text:
+  en: |
+    Hello {username}!
+
+    User {ownerName} has ended your membership to room {roomName}.
```

Comment:
According to my research it would be typical to say "membership **in**"

Reply:
I trust your research, updated.